include/RAJA/policy/hip/policy.hpp

```diff
@@ -345,6 +499,43 @@ template < bool with_atomic >
 using hip_reduce_base = std::conditional_t<with_atomic, hip_reduce_atomic, hip_reduce>;
 
 
+template < RAJA::hip::multi_reduce_algorithm algorithm,
+           typename SharedAtomicReplicationConcretizer,
+           typename GlobalAtomicReplicationConcretizer >
+using hip_multi_reduce_tuning = hip_multi_reduce_policy<
+    RAJA::hip::MultiReduceTuning<algorithm,
+                                 SharedAtomicReplicationConcretizer,
+                                 GlobalAtomicReplicationConcretizer>>;
+
+// Policies for RAJA::MultiReduce* objects with specific behaviors.
+// - *atomic* policies may use atomics to combine partial results. The
+//   use of atomics leads to order of operation differences which change the
+//   results of floating point sum reductions run to run.
+// - *host_init* policies are similar to the atomic policies above. However
+//   the memory used with atomics is initialized on the host which is
+//   significantly cheaper on some HW. On some HW this is faster overall than
```

Comment:
It would be helpful to note which HW is is faster.

Reply:
updated